modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -178,6 +178,22 @@
 			"jspf"
 		]
 	},
+	{
+		"from": "CookieUtil.get(HttpServletRequest, String)",
+		"issueKey": "LPS-159681",
```

Comment:
@MicaelleSilvaa, `issueKey` not sorted correctly, can you please fix?

Reply:
Resolved :)